plugins/infinispan80/src/main/java/org/radargun/stream/example/WordCount.java

```diff
@@ -0,0 +1,44 @@
+package org.radargun.stream.example;
+
+import org.infinispan.stream.CacheCollectors;
+import org.radargun.traits.Streamable;
+
+import java.io.Serializable;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * @author Roman Macor &lt;rmacor@redhat.com&gt;
+ */
+public class WordCount implements Streamable.StreamFunction {
+
+   private Object result;
+
+   @Override
+   public Object apply(Stream stream) {
+      result = stream.map((Serializable & Function<Map.Entry<String, String>, String[]>) e -> e.getValue().split("[\\p{Punct}\\s&&[^'-]]+")).
+              flatMap((Serializable & Function<String[], Stream<String>>) Arrays::stream).
+              collect(CacheCollectors.serializableCollector(() -> Collectors.groupingBy(Function.identity(), Collectors.counting())));
+      return result;
+   }
+
+   @Override
+   public String getPrintableResult() {
+      Map<Object, Object> resultMap = (Map<Object, Object>) result;
+      StringBuilder sb = new StringBuilder();
+      for (Map.Entry<Object, Object> entry : resultMap.entrySet()) {
+         sb.append(System.getProperty("line.separator"));
```

Comment:
@mcimbora and I discussed it, the output looks better in logs if it starts with an empty line.